models/taskRequests.js

```diff
@@ -39,6 +41,23 @@ const fetchTaskRequests = async () => {
   return taskRequests;
 };
 
+/**
+ * Fetches task request by id
+ *
+ * @param taskRequestId { string }: id of task request
+ * @return Promise<{taskRequest: Object}>
+ */
+const fetchTaskRequestById = async (taskRequestId) => {
+  const taskRequestSnapshot = await taskRequestsCollection.doc(taskRequestId).get();
+  const taskRequestData = taskRequestSnapshot.data();
+
+  if (taskRequestData) {
+    taskRequestData.id = taskRequestSnapshot.id;
+    taskRequestData.url = new URL(`/taskRequests/${taskRequestData.id}`, config.get("services.rdsUi.baseUrl"));
+  }
+  return taskRequestData;
+};
+
```

Comment:
Why is there no try catch?

Reply:
Currently there is no logger for our backend and we're just throwing the error in the catch statement, so having a catch here will be redundant as we're already catching it in the controller.